nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -24,6 +60,224 @@ in
+          description = lib.mdDoc ''
+            Additional bluetooth configuration for the WirePlumber daemon when
+            run in multi-instance mode. By default the WirePlumber daemon runs in
+            single-instance mode with nixpkgs, meaning this is ignored by default.
+            See `services.pipewire.wireplumber.extraConfig.wireplumber`.
+
+            The provided config lines are appended to the default
+            `''${pkgs.wireplumber}/share/wireplumber/bluetooth.conf`.
+
+            ${configUnwantedUnsettingWarning}
+
+            See also:
+            - [Running Multiple Instances][docs-config-multi-instance]
+            - [Bluetooth Configuration][docs-config-bluetooth]
+            - [Location of Files][docs-config-file-locations]
+
+            [docs-config-file-locations]: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/0.4/docs/rst/configuration/locations.rst
+            [docs-config-bluetooth]: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/0.4/docs/rst/configuration/bluetooth.rst
+            [docs-config-multi-instance]: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/0.4/docs/rst/configuration/multi_instance.rst
+          '';
+        };
+        policy = lib.mkOption {
+          type = lib.types.lines;
+          default = "";
+          description = lib.mdDoc ''
+            Additional policy configuration for the WirePlumber daemon when
+            run in multi-instance mode. By default the WirePlumber daemon runs in
+            single-instance mode with nixpkgs, meaning this is ignored by default.
+            See `services.pipewire.wireplumber.extraConfig.wireplumber`.
+
+            The provided config lines are appended to the default
+            `''${pkgs.wireplumber}/share/wireplumber/policy.conf`.
+
+            ${configUnwantedUnsettingWarning}
+
+            See also:
+            - [Running Multiple Instances][docs-config-multi-instance]
+            - [Policy Configuration][docs-config-policy]
+            - [Location of Files][docs-config-file-locations]
+
+            [docs-config-file-locations]: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/0.4/docs/rst/configuration/locations.rst
+            [docs-config-policy]: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/0.4/docs/rst/configuration/policy.rst
+            [docs-config-multi-instance]: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/0.4/docs/rst/configuration/multi_instance.rst
+          '';
+        };
+      };
+
+      extraLuaConfig = {
+        main = lib.mkOption {
+          type = lib.types.attrsOf lib.types.lines;
+          default = { };
+          example = {
+            "41-celluloid-default-volume" = ''
+              table.insert (stream_defaults.rules, {
+                {
+                  matches = {
+                    {
+                      { "application.name", "matches", "io.github.celluloid_player.Celluloid" },
```

Comment:
This one came easily and I'm rather happy with it as well. I'll try and see if I can find similarly good examples for the others as well

Reply:
I've added an example to the bluetooth and policy Lua config each. I gotta note that I've not tested these examples. I might try to today or in the next few days, but especially the policy config seems a bit hard to test since I know of nothing that actually uses policy defined endpoints at the moment (in the usual desktop setup of wireplumber that I have). Then again, maybe I just need to look harder.